yt/visualization/profile_plotter.py

```diff
@@ -1336,77 +1337,6 @@ def save(self, name=None, suffix=".png", mpl_kwargs=None):
             self.plots[f].save(name, mpl_kwargs)
         return names
 
-    @invalidate_plot
-    def set_font(self, font_dict=None):
```

Comment:
Wait, so can we no longer manually set the font in our plots? I don't think i ever have, but it might be something people have used in the past.

Reply:
No, it will still be possible, thanks to inheritance. Because I'm making `ProfilePlot` a subclass of `PlotContainer`, we don't need to have this method here, which was duplicated code from `PlotContainer` anyway :-)